mmedit/datasets/pipelines/generate_assistant.py

```diff
@@ -167,3 +167,75 @@ def __repr__(self):
         repr_str += (f'sample_quantity={self.sample_quantity}, '
                      f'scale={self.scale}, target_size={self.target_size}')
         return repr_str
+
+
+@PIPELINES.register_module()
+class GenerateLandmark:
+    """Generate face landmark.
+
+    Args:
+        image_key (str): Key of face image.
+        device (str): Device of face alignment. Default: 'cpu'.
+    """
+
+    def __init__(self, image_key, device='cpu'):
+        self.image_key = image_key
+        self.face_alignment_model = None
+        self.init = False
+        self.device = device
+
+    def __call__(self, results):
+        """Call function.
+
+        Args:
+            results (dict): A dict containing the necessary information and
+                data for augmentation. Require image_key.
+
+        Returns:
+            dict: A dict containing the processed data and information.
+                Add 'landmark'.
+        """
+
+        image = results[self.image_key]
+        landmark = self._face_alignment_detector(image)
+        results['landmark'] = landmark
+
+        return results
+
+    def _face_alignment_detector(self, image):
+        """Generate face landmark by face_alignment.
+
+        Args:
+            image (numpy.ndarray): Face image.
+
+        Returns:
+            landmark (Tuple[float]): Location of landmark.
+        """
+
+        try:
+            import face_alignment
```

Comment:
This line would be called whenever `_face_alignment_detector` is called. 

Suggest to import only once as in [here](https://github.com/open-mmlab/mmediting/blob/master/mmedit/apis/restoration_face_inference.py#L8).